docs/quickstart.md

```diff
@@ -118,39 +99,46 @@ func main() {
 
     ctx := context.Background()
 
-    // create a user
-    createdUser, err := client.User.CreateOne(
-        db.User.Email.Set("john.doe@example.com"),
-        db.User.Name.Set("John Doe"),
-
-        // ID is optional, which is why it's specified last. if you don't set it
-        // an ID is auto generated for you
-        db.User.ID.Set("123"),
+    // create a post
+    createdPost, err := client.Post.CreateOne(
+        db.Post.Title.Set("Hi from Prisma!"),
+        db.Post.Published.Set(true),
+        db.Post.Desc.Set("Prisma is a database toolkit and makes databases easy."),
+        // ID is optional since it's auto generated, which is why it's specified last.
+        db.Post.ID.Set("123"),
     ).Exec(ctx)
+    if err != nil {
+        panic(err)
```

Comment:
I think we should create a small `func run() error` that main calls to avoid seeing these panics everywhere.

Reply:
do you mean we should just `return err` instead?

Reply:
or are you talking about a function to reduce 3 lines of if&return to a `panicIf(err)` method?

Reply:
yah